src/services/montage-helper.service.js

```diff
@@ -5,7 +5,27 @@
 		.module('montage')
 		.factory('montageHelper', montageHelper);
 
-	function montageHelper(montage, authService) {
+	function montageHelper(montage, authService, ngProgressFactory) {
+
+		var count = 0;
+		var progress = ngProgressFactory.createInstance();
+		progress.setColor('#01579B');
```

Comment:
Create a file named `config/constants.config.js` and add the color code as an app constant.